python/hail/expr/functions.py

```diff
@@ -2029,6 +2029,10 @@ def gt_from_pl(pl):
     """
     return _func("gtFromPL", TInt32(), pl)
 
+@typecheck(gt=expr_call, i=expr_int32)
+def downcode(gt, i):
+    return call(cond(gt.gtj() == i, 1, 0) + cond(gt.gtk() == i, 1, 0))
```

Comment:
Should probably write a short docstring for this.

Reply:
done.